src/compat.lisp

```diff
@@ -41,3 +41,11 @@
         :until (terminalp ch)))
 
 (set-dispatch-macro-character #\# #\J #'j-reader)
+
```

Comment:
I think `get-source-data-epoch` should only be concerned with the environment variable.  Can we make it return `nil` if it is not present?

Also, if it is not `ccl` or `sbcl` we can just signal an error.

Later on, in `toplevel.lisp`, we can do 

```lisp
(or (get-soruce-data-epoch)
    (get-universal-time))
```

Reply:
I agree.
It will take some time.

Reply:
The only thing that is not sure is that need to initiate an error in this case.

Reply:
Something like

```lisp
#-(sbcl ccl) (error "This Lisp implementation is not supported yet!")
```

?

Reply:
I'm not sure about the need for such an exception. 
The function will return NIL and execution will continue.
It is possible that it is necessary to write in the release note  `is tested on CCL/SBCL compilers only`

Reply:
In my opinion it will be a little unnatural exception.

Reply:
Ok. That's also fine.